VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitVideoFragment.java

```diff
@@ -75,19 +69,26 @@
     private final Handler playHandler = new Handler();
     private View messageContainer;
 
+    private final static String HAS_NEXT_UNIT_ID = "has next unit";
```

Comment:
Use underscores for stringdefs. I don't know what the official AOSP style is, but we should at least keep it consistent with our other stringdefs.